Godeps/Godeps.json

```diff
@@ -104,11 +104,13 @@
 		},
 		{
 			"ImportPath": "github.com/golang/mock/gomock",
-			"Rev": "06883d979f10cc178f2716846215c8cf90f9e363"
+			"Comment": "v1.0.0-17-g18f6dd1",
+			"Rev": "18f6dd13ccdd227b8ebc546ca95cd62a02f3970c"
```

Comment:
Can you add the unit test output for the `github.com/golang/mock/gomock` package to the PR since its being updated?

Reply:
Done.